### DISCUSSION TITLE
Enjoying Turbo without linking

### DISCUSSION THREAD

**Discussion by goldbergyoni** [vercel/turborepo#1581] [Help] [answered]
I'm evaluating Turbo as the default monorepo for [practica.js](https://github.com/practicajs/practica): We have multiple packages and wish to avoid local linking between them, they consume each other via npm registry

On top of that, I wish to enjoy the great Turbo features like caching, parallel execution and more

To my understanding, I must use a workspace underneath Turbo and those worksapces by default link local packages

Is there a way for me to tell Turbo 'those are my packages', just run and cache tasks without dependencies?

**Comment by anthonyshew** [comment, marked as answer]
Yes, you are able to publish packages and use them in your monorepo.
